src/libpriv/rpmostree-postprocess.cxx

```diff
@@ -708,13 +703,13 @@ filter_xattrs_impl (OstreeRepo     *repo,
   if (!*relpath)
     {
       if (!glnx_fd_get_all_xattrs (rootfs_fd, &existing_xattrs, NULL, error))
-        util::throw_gerror(local_error);
+        g_error ("Reading xattrs on /: %s", local_error->message);
```

Comment:
This is an interesting one, because we can't unwind up properly anymore.  But eh; it's a bug in the ostree API, we should have falliable versions of these callbacks.

Reply:
Yeah. I changed it to a `g_error` because we were exiting immediately anyway where that exception was caught in `filter_xattrs_cb`.